assignment-1/Program.cs

```diff
@@ -0,0 +1,274 @@
+﻿using System;
+using System.Collections.Generic;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Library library = new Library();
+
+        bool exit = false;
+
+        Console.WriteLine("Welcome to the Library Management System");
+        while (!exit)
+        {
+            Console.WriteLine("\n1. Create Admin Account");
+            Console.WriteLine("2. Add Book");
+            Console.WriteLine("3. Register User");
+            Console.WriteLine("4. Issue Book");
+            Console.WriteLine("5. Return Book");
+            Console.WriteLine("6. List books issued by User");
+            Console.WriteLine("7. Get book inventory info");
+            Console.WriteLine("8. List users with the book");
+            Console.WriteLine("9. Calculate fine for user");
+            Console.WriteLine("10. Manage Inventory (Increase)");
+            Console.WriteLine("11. Manage Inventory (Reduce)");
+            Console.WriteLine("12. Exit\n");

```

Comment:
Instead of directly starting with all the options what should be ideal behaviour is ask whether to continue as a Admin or Student or Teacher and then you can confirm the identity using some authentication (or username if you don't want to have the authentication) and then only show the options based on the role of logged in User.